lib/peep.rb

```diff
@@ -0,0 +1,39 @@
+require 'PG'
+
+class Peep
+
+  attr_reader :id, :message, :time
+
+  def initialize(id:, message:, time:)
+    @id = id
+    @message = message
+    @time = time
+  end
+
+  def self.show
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps ORDER BY id DESC;")
```

Comment:
I hadn't thought of sorting by ID, that's clever as ID is generated sequentially!